### PR TITLE
ADDED USER ROUTES

### PR DESCRIPTION
**Pull request by khanrabia**
Need a Review

### CODE REVIEW

File: routes/userRouter.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+router.get('/user/:id/events', userController.getUserEvents);
+
+// GET: /user/:id/reservations
+router.get('/user/:id/reservations', userController.getUserReservations);
+
+// POST: /newUser
+router.post('/newUser', userController.createUser);
+
+// PATCH: /user/:id (updateUser)
+router.put('/user/:id', userController.updateUser);
```
Comment (ShemarG) [inline routes/userRouter.js:18]: More of a stylistic convention, but usually RESTful routes are plural, so `user` would be `users` .

**Review by ShemarG — CHANGES_REQUESTED** [review]
Everything for the most part looks great. Do see the comments I left

File: db/init.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+DROP TABLE IF EXISTS Users;
+
+CREATE TABLE Users(
+	id SERIAL PRIMARY KEY,
+	email TEXT,
+	username TEXT,
+	password TEXT,
+	zip TEXT,
+	profile_pic BYTEA,
+	favorite_tags TEXT
```
Comment (Ricardo-21) [inline db/init.sql:12]: We got rid of this the previous commit, best to remove this to avoid merge conflicts

File: db/init.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+CREATE TABLE Events(
+	id SERIAL PRIMARY KEY,
+	title TEXT,
+	user_id INT REFERENCES Users(id),
+	location TEXT,
+	time_start TEXT,
+	time_end TEXT,
+	isVirtual BOOL,
+	tags TEXT
```
Comment (Ricardo-21) [inline db/init.sql:23]: We may be able to resolve these conflicts here on github, if possible remove line 23 and BOOL should be BOOLEAN

**Review by Ricardo-21 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**